src/Umbraco.Core/Extensions/PublishedContentExtensions.cs

```diff
@@ -1295,134 +1295,6 @@ public static string GetWriterName(this IPublishedContent content, IUserService
 
     #endregion
 
-    #region Axes: children
-
-    /// <summary>
-    ///     Gets the children of the content in a DataTable.
-    /// </summary>
-    /// <param name="content">The content.</param>
-    /// <param name="variationContextAccessor">Variation context accessor.</param>
-    /// <param name="contentTypeService">The content type service.</param>
-    /// <param name="mediaTypeService">The media type service.</param>
-    /// <param name="memberTypeService">The member type service.</param>
-    /// <param name="publishedUrlProvider">The published url provider.</param>
-    /// <param name="contentTypeAliasFilter">An optional content type alias.</param>
-    /// <param name="culture">
-    ///     The specific culture to filter for. If null is used the current culture is used. (Default is
-    ///     null)
-    /// </param>
-    /// <returns>The children of the content.</returns>
-    public static DataTable ChildrenAsTable(
```

Comment:
Similar to the `DataTableExtensions`, this is legacy code that isn't used anymore.